### DISCUSSION TITLE
Welcome to python-audio-separator Discussions!

### DISCUSSION THREAD

**Discussion by beveradb** [nomadkaraoke/python-audio-separator#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zhyqieqie** [comment]
I need to implement the function of separating audio in Python, as many require voice separation on web pages, and I am willing to incorporate these practices into the program to simplify the separation process.
